examples/analog/main.cpp

```diff
@@ -54,7 +54,6 @@ int main(int argc, char* argv[])
 	trig->setAnalogDelay(0);
```

Comment:
this commit looks funky
it's only the `Signed-off-by` tag
maybe squash everything into the first one?